read_lyrics.py

```diff
@@ -0,0 +1,105 @@
+import pronouncing
+
+def analyze_text(myString):
+    myString = myString.replace(",", "")
+    myString = myString.replace(".", "")
+    myString = myString.replace("?", "")
+    myList = myString.split(" ")
+    running = []
+    running.append(myList[0])
+
+    idk = 0
+    same = 0
+    usedRhymes = []
+    for i in range(1, len(myList)):
+        for j in running:
+            if not usedRhymes.__contains__(j + myList[i]):
+                usedRhymes.append(j + myList[i])
+                rating = two_rhyme(j, myList[i])
+                if rating == -1:
+                    while two_rhyme(j, myList[i]) == -1 and len(myList[i]) > 1:
+                        myList[i] = myList[i][1:len(myList[i])]
+                    if two_rhyme(j, myList[i]) == -1:
```

Comment:
This is pretty complicated logic here so in-line comment describing what is happening would be good.